scheds/rust/scx_layered/src/main.rs

```diff
@@ -900,6 +901,12 @@ struct Layer {
 impl Layer {
     fn new(spec: &LayerSpec, idx: usize, cpu_pool: &CpuPool, topo: &Topology) -> Result<Self> {
         let name = &spec.name;
+        assert!(
+            name.len() < MAX_LAYER_NAME,
+            "layer {} name too long,  max {}",
+            name,
+            MAX_LAYER_NAME
+        );
```

Comment:
Do we want to do this or trim the string instead?

Reply:
I guess that's probably not worth blowing up for since it's debugging info.